src/tools/dev/scripts/bv_support/bv_uintah.sh

```diff
@@ -269,7 +269,7 @@ function build_uintah
         info "Invoking command to configure UINTAH"
         set -x
         sh -c "../src/configure CXX=\"$CXX_COMPILER\" CC=\"$C_COMPILER\" \
-        CFLAGS=\"$CFLAGS $C_OPT_FLAGS -headerpad_max_install_names\" CXXFLAGS=\"$CXXFLAGS $CXX_OPT_FLAGS\" \
+        CFLAGS=\"$CFLAGS $C_OPT_FLAGS -std=c++11 -headerpad_max_install_names\" CXXFLAGS=\"$CXXFLAGS $CXX_OPT_FLAGS -std=c++11\" \
```

Comment:
Does this new flag need to be in `CFLAGS` too or only in `CXXFLAGS`? Because its specifying something about the C++ standard, my first thought is that it would go into only `CXXFLAGS`.

Reply:
I was  following the instructions from the discussion, because customer found the solution to uintah not building on ubuntu22.

Do the -std flags apply on mac?

Reply:
> I was following the instructions from the discussion, because customer found the solution to uintah not building on ubuntu22.

But, do we know for sure that the original solution actually required changes to both `CFLAGS` and `CXXFLAGS`? It just doesn't seem correct to me to specify a C++ standard in `CFLAGS` because `CFLAGS` is for the C compiler and specifying a C++ standard to the C compiler seems wrong...maybe harmless but still wrong.

> Do the -std flags apply on mac?

Yes.

Reply:
@markcmiller86 I removed '-std=c++11' from CFLAGS.